donation-ticker.html

```diff
@@ -27,6 +27,7 @@ <h1 class="h5">{{ entity.sumDonations | formatMoney }} / {{ entity.fundraisingGo
 								<th scope="col">$</th>
 								<th scope="col">Name</th>
 								<th scope="col">Message</th>
+								<th scope="col">Incentive</th>
```

Comment:
i think i said it'd be "nice to have," but this column should either display based on configuration, OR if the fundraiser actually has incentives